src/test/scala/org/zalando/kanadi/PekkoTestKitBase.scala

```diff
@@ -0,0 +1,34 @@
+package org.zalando.kanadi
+
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.http.scaladsl.{Http, HttpExt}
+import org.apache.pekko.testkit.{TestKit, TestKitBase}
+import org.scalactic.Prettifier
+import org.scalatest.{BeforeAndAfterAll, FixtureTestSuite, TestData, fixture}
+
+import scala.concurrent.{Await, ExecutionContext}
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+trait PekkoTestKitBase extends TestKitBase with BeforeAndAfterAll with fixture.TestDataFixture {
+  this: FixtureTestSuite =>
+  implicit val system: ActorSystem
+
+  implicit lazy val http: HttpExt = Http()
+
+  implicit lazy val executionContext: ExecutionContext = system.dispatcher
+  override protected def afterAll(): Unit = {
+    val future = http.shutdownAllConnectionPools().map(_ => TestKit.shutdownActorSystem(system))(system.dispatcher)
+    Await.result(future, 10 seconds)
+  }
+
+  def pp(obj: Any)(implicit testData: TestData = null) = {
```

Comment:
The `= null` trick can be used to detect whether an implicit `TestData` is in scope or not.